.travis.yml

```diff
@@ -0,0 +1,15 @@
+os: osx
+language: objective-c
+osx_image: xcode8.3
+node_js: 5.1.1
+
+before_install:
+  - cd server
+
+install:
+  - npm install
+
+script:
```

Comment:
cool, I wouldn't worry about it for now - hardy any test for the native part of the app anyway 😳